src/mux-analytics.brs

```diff
@@ -212,6 +212,8 @@ function runBeaconLoop()
           m.mxa.rebufferStartHandler()
         else if field = "rebufferend"
           m.mxa.rebufferEndHandler()
+        else if field = "playback_mode"
+          m.mxa.playbackModeHandler()
```

Comment:
don't you need to pass in the value you read from the message? This is just going to make a call to the sub without any param, so... how does this work?